### ISSUE TITLE
French localization

### ISSUE THREAD

**Issue by aidenmitchell** [OrangeJedi/Aerial#22]
Is there a way to localize this? I can help with French localization. Also, thank you for continuing the Windows Aerial, I'm glad this is being maintained.

**Comment by Aleatoire** [comment]
+1

**Comment by OrangeJedi** [comment]
Certainly! This is something I was planning on doing if it seemed there was an interest in it.

 I know Apple publishes translations of the video names/descriptions with the videos so we are covered there. I will need help with any other translations though

I'll look into what this will take and my end and post my progress here.

**Comment by aidenmitchell** [comment]
Sounds great, thanks!

**Comment by MyRessources** [comment]
Hi, i discover this project today but i can help to translate it in french if you want :)
